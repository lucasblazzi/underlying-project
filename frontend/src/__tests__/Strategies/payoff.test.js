import React from 'react';
import Payoff from '../../pages/Strategies/payoff';
import renderer from 'react-test-renderer';
import ReactApexChart from "react-apexcharts";

it('PÃ¡gina', () => {
    const tree = renderer.create(<div id="line-chart"/>).toJSON();
    expect(tree).toMatchSnapshot();
})