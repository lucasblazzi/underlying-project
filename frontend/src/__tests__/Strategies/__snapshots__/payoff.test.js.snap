// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PÃ¡gina 1`] = `
<div
  id="line-chart"
/>
`;
